compute/admin_guide/install/system_requirements.adoc

```diff
@@ -81,7 +81,7 @@ Prisma Cloud has been tested on the following services:
 * Microsoft Azure
 * Oracle Cloud Infrastructure (OCI)
 
-=== Architecture: ARM64
+=== ARM Architecture-specific Requirements
```

Comment:
Do we need the word specific? Would calling it ARM Architecture Requirements work?
If yes please make it Specific with "S".

And for the lead in sentence, please consider replacing 
To use Prisma Cloud on ARM architecture, `consider ` the following requirements with `adhere to` or `review`